[2023-10-08T02:43:00.594+0000] {processor.py:157} INFO - Started process (PID=37715) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:00.600+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:43:00.606+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:43:00.604+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:00.677+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:43:00.669+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:43:00.682+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:00.774+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.214 seconds
[2023-10-08T02:43:31.717+0000] {processor.py:157} INFO - Started process (PID=37781) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:31.721+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:43:31.724+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:43:31.723+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:31.744+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:43:31.740+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:43:31.745+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:43:31.788+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.080 seconds
[2023-10-08T02:44:02.488+0000] {processor.py:157} INFO - Started process (PID=37860) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:02.491+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:44:02.494+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:02.493+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:02.509+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:02.505+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:44:02.511+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:02.545+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.065 seconds
[2023-10-08T02:44:24.264+0000] {processor.py:157} INFO - Started process (PID=37927) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:24.267+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:44:24.270+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:24.269+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:24.291+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:24.288+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:44:24.293+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:24.328+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.069 seconds
[2023-10-08T02:44:27.456+0000] {processor.py:157} INFO - Started process (PID=37935) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:27.458+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:44:27.460+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:27.459+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:27.494+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:27.491+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:44:27.495+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:27.524+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.073 seconds
[2023-10-08T02:44:28.489+0000] {processor.py:157} INFO - Started process (PID=37936) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:28.491+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:44:28.493+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:28.493+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:28.506+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:28.503+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:44:28.507+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:28.540+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.059 seconds
[2023-10-08T02:44:59.126+0000] {processor.py:157} INFO - Started process (PID=38023) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:59.129+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:44:59.131+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:59.131+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:59.145+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:44:59.141+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:44:59.146+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:44:59.178+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.057 seconds
[2023-10-08T02:45:29.427+0000] {processor.py:157} INFO - Started process (PID=38101) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:29.438+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:45:29.440+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:45:29.440+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:29.459+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:45:29.454+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:45:29.461+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:29.513+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.094 seconds
[2023-10-08T02:45:59.623+0000] {processor.py:157} INFO - Started process (PID=38179) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:59.627+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:45:59.631+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:45:59.630+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:59.656+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:45:59.648+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:45:59.658+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:45:59.719+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.105 seconds
[2023-10-08T02:46:30.197+0000] {processor.py:157} INFO - Started process (PID=38256) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:46:30.201+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:46:30.205+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:46:30.204+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:46:30.245+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:46:30.220+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:46:30.249+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:46:30.311+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.123 seconds
[2023-10-08T02:47:00.895+0000] {processor.py:157} INFO - Started process (PID=38334) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:00.900+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:47:00.903+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:47:00.902+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:00.921+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:47:00.916+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:47:00.923+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:00.973+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.088 seconds
[2023-10-08T02:47:31.080+0000] {processor.py:157} INFO - Started process (PID=38412) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:31.086+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:47:31.092+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:47:31.091+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:31.124+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:47:31.114+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:47:31.127+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:47:31.210+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.147 seconds
[2023-10-08T02:48:02.378+0000] {processor.py:157} INFO - Started process (PID=38490) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:02.397+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:02.441+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:02.438+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:02.517+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:02.509+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:48:02.526+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:02.624+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.264 seconds
[2023-10-08T02:48:23.156+0000] {processor.py:157} INFO - Started process (PID=38539) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:23.159+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:23.165+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:23.164+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:23.198+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:23.192+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:48:23.218+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:23.292+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.146 seconds
[2023-10-08T02:48:31.624+0000] {processor.py:157} INFO - Started process (PID=38559) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:31.629+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:31.634+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:31.633+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:31.656+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:31.653+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 10
    from
        ^
SyntaxError: invalid syntax
[2023-10-08T02:48:31.658+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:31.726+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.116 seconds
[2023-10-08T02:48:38.938+0000] {processor.py:157} INFO - Started process (PID=38580) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:38.941+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:38.944+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:38.943+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:38.959+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:38.955+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 10
    from group_mapping_layer import mapping_task()
                                                ^
SyntaxError: invalid syntax
[2023-10-08T02:48:38.961+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:39.008+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.079 seconds
[2023-10-08T02:48:41.385+0000] {processor.py:157} INFO - Started process (PID=38584) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:41.388+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:41.391+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:41.390+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:41.404+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:41.402+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 10
    from group_mapping_layer import mapping_tasks()
                                                 ^
SyntaxError: invalid syntax
[2023-10-08T02:48:41.406+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:41.462+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.086 seconds
[2023-10-08T02:48:44.217+0000] {processor.py:157} INFO - Started process (PID=38592) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:44.220+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:44.224+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:44.223+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:44.240+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:44.237+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 10
    from group_mapping_layer import mapping_tasks()
                                                 ^
SyntaxError: invalid syntax
[2023-10-08T02:48:44.242+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:44.306+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.097 seconds
[2023-10-08T02:48:47.889+0000] {processor.py:157} INFO - Started process (PID=38605) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:47.892+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:47.895+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:47.894+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:47.934+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:47.924+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:48:47.948+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:48.003+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.121 seconds
[2023-10-08T02:48:49.605+0000] {processor.py:157} INFO - Started process (PID=38619) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:49.607+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:49.610+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:49.609+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:49.625+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:49.622+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11
    from
        ^
SyntaxError: invalid syntax
[2023-10-08T02:48:49.627+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:49.673+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.075 seconds
[2023-10-08T02:48:54.072+0000] {processor.py:157} INFO - Started process (PID=38625) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:54.075+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:54.077+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:54.077+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:54.092+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:54.089+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11
    from group_staging_layer import
                                   ^
SyntaxError: invalid syntax
[2023-10-08T02:48:54.094+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:54.136+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.071 seconds
[2023-10-08T02:48:58.498+0000] {processor.py:157} INFO - Started process (PID=38631) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:58.502+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:58.505+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:58.504+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:58.530+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:58.526+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:48:58.533+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:58.575+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.085 seconds
[2023-10-08T02:48:59.643+0000] {processor.py:157} INFO - Started process (PID=38634) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:59.646+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:48:59.649+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:59.649+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:59.680+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:48:59.674+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import group_mapping_layer
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:48:59.682+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:48:59.744+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.110 seconds
[2023-10-08T02:49:26.135+0000] {processor.py:157} INFO - Started process (PID=38709) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:26.138+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:49:26.141+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:26.140+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:26.173+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:26.168+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import groups
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:49:26.174+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:26.245+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.118 seconds
[2023-10-08T02:49:27.161+0000] {processor.py:157} INFO - Started process (PID=38710) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:27.166+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:49:27.170+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:27.169+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:27.203+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:27.197+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import groups
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:49:27.205+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:27.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.106 seconds
[2023-10-08T02:49:31.793+0000] {processor.py:157} INFO - Started process (PID=38732) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:31.797+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:49:31.800+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:31.799+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:31.832+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:49:31.826+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import groups
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:49:31.835+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:49:31.938+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.153 seconds
[2023-10-08T02:50:02.766+0000] {processor.py:157} INFO - Started process (PID=38803) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:02.771+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:02.774+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:02.773+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:02.796+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:02.790+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8, in <module>
    from dags import groups
ModuleNotFoundError: No module named 'dags'
[2023-10-08T02:50:02.799+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:02.867+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.111 seconds
[2023-10-08T02:50:11.702+0000] {processor.py:157} INFO - Started process (PID=38828) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:11.709+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:11.715+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:11.714+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:11.736+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:11.731+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.
               ^
SyntaxError: invalid syntax
[2023-10-08T02:50:11.740+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:11.796+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.103 seconds
[2023-10-08T02:50:14.485+0000] {processor.py:157} INFO - Started process (PID=38837) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:14.487+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:14.490+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:14.490+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:14.506+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:14.503+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.group_mapping_layer
                                  ^
SyntaxError: invalid syntax
[2023-10-08T02:50:14.508+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:14.552+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.074 seconds
[2023-10-08T02:50:19.239+0000] {processor.py:157} INFO - Started process (PID=38851) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:19.243+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:19.249+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:19.247+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:19.275+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:19.271+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.group_mapping_layer import
                                          ^
SyntaxError: invalid syntax
[2023-10-08T02:50:19.279+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:19.407+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.189 seconds
[2023-10-08T02:50:29.299+0000] {processor.py:157} INFO - Started process (PID=38875) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:29.305+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:29.310+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:29.309+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:29.372+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:29.362+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 10, in <module>
    from group_mapping_layer import mapping_tasks
ModuleNotFoundError: No module named 'group_mapping_layer'
[2023-10-08T02:50:29.377+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:29.429+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.145 seconds
[2023-10-08T02:50:34.106+0000] {processor.py:157} INFO - Started process (PID=38888) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:34.118+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:34.123+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:34.123+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:34.152+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:34.148+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 9
    from groups.g
                ^
SyntaxError: invalid syntax
[2023-10-08T02:50:34.156+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:34.231+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.146 seconds
[2023-10-08T02:50:36.457+0000] {processor.py:157} INFO - Started process (PID=38905) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:36.462+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:36.469+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:36.467+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:36.598+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:36.590+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 9
    from groups.group_staging_layer import
                                         ^
SyntaxError: invalid syntax
[2023-10-08T02:50:36.602+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:36.789+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.357 seconds
[2023-10-08T02:50:39.443+0000] {processor.py:157} INFO - Started process (PID=38910) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:39.450+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:50:39.462+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:39.460+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:39.519+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:50:39.514+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 9, in <module>
    from groups.group_staging_layer import staging_tasks
ImportError: cannot import name 'staging_tasks' from 'groups.group_staging_layer' (/opt/airflow/dags/groups/group_staging_layer.py)
[2023-10-08T02:50:39.522+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:50:39.577+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.147 seconds
[2023-10-08T02:51:12.539+0000] {processor.py:157} INFO - Started process (PID=38993) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:12.548+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:51:12.556+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:51:12.554+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:12.634+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:51:12.623+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 9, in <module>
    from groups.group_staging_layer import staging_tasks
  File "/opt/airflow/dags/groups/group_staging_layer.py", line 10
    with TaskGroup("staging_layer", tooltip='Mapping layer tasks') as group:
       ^
IndentationError: expected an indented block
[2023-10-08T02:51:12.637+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:12.778+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.262 seconds
[2023-10-08T02:51:43.637+0000] {processor.py:157} INFO - Started process (PID=39064) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:43.645+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:51:43.652+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:51:43.650+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:43.739+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:51:43.727+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11, in <module>
    from group_mapping_layer import mapping_tasks
ModuleNotFoundError: No module named 'group_mapping_layer'
[2023-10-08T02:51:43.743+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:51:43.873+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.261 seconds
[2023-10-08T02:52:14.607+0000] {processor.py:157} INFO - Started process (PID=39137) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:14.610+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:52:14.613+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:14.612+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:14.641+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:14.636+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11, in <module>
    from group_mapping_layer import mapping_tasks
ModuleNotFoundError: No module named 'group_mapping_layer'
[2023-10-08T02:52:14.643+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:14.690+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.092 seconds
[2023-10-08T02:52:45.430+0000] {processor.py:157} INFO - Started process (PID=39219) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:45.438+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:52:45.443+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:45.442+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:45.481+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:45.477+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11, in <module>
    from group_mapping_layer import mapping_tasks
ModuleNotFoundError: No module named 'group_mapping_layer'
[2023-10-08T02:52:45.483+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:45.543+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.120 seconds
[2023-10-08T02:52:57.813+0000] {processor.py:157} INFO - Started process (PID=39265) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:57.816+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:52:57.818+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:57.818+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:57.831+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:57.828+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.
               ^
SyntaxError: invalid syntax
[2023-10-08T02:52:57.832+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:57.880+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.077 seconds
[2023-10-08T02:52:59.976+0000] {processor.py:157} INFO - Started process (PID=39268) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:52:59.981+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:52:59.986+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:52:59.985+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:00.007+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:00.002+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.group_mapping_layer
                                  ^
SyntaxError: invalid syntax
[2023-10-08T02:53:00.010+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:00.057+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.091 seconds
[2023-10-08T02:53:01.721+0000] {processor.py:157} INFO - Started process (PID=39281) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:01.724+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:53:01.727+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:01.726+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:01.742+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:01.738+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 8
    from groups.group_mapping_layer import
                                          ^
SyntaxError: invalid syntax
[2023-10-08T02:53:01.744+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:01.795+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.084 seconds
[2023-10-08T02:53:10.764+0000] {processor.py:157} INFO - Started process (PID=39296) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:10.768+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:53:10.771+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:10.770+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:10.812+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:10.807+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 11, in <module>
    from group_mapping_layer import mapping_tasks
ModuleNotFoundError: No module named 'group_mapping_layer'
[2023-10-08T02:53:10.814+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:10.862+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.105 seconds
[2023-10-08T02:53:12.186+0000] {processor.py:157} INFO - Started process (PID=39299) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:12.189+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:53:12.192+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.191+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:12.240+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:12.503+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.502+0000] {manager.py:504} INFO - Created Permission View: can edit on DAG:group_dag
[2023-10-08T02:53:12.524+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.523+0000] {manager.py:504} INFO - Created Permission View: can read on DAG:group_dag
[2023-10-08T02:53:12.539+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.538+0000] {manager.py:504} INFO - Created Permission View: can delete on DAG:group_dag
[2023-10-08T02:53:12.567+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.567+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:53:12.596+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.596+0000] {dag.py:2763} INFO - Creating ORM DAG for group_dag
[2023-10-08T02:53:12.803+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:12.803+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:53:12.844+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.666 seconds
[2023-10-08T02:53:41.525+0000] {processor.py:157} INFO - Started process (PID=39377) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:41.533+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:53:41.547+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:41.545+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:41.654+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:53:41.695+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:41.694+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:53:41.787+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:53:41.786+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:53:41.853+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.345 seconds
[2023-10-08T02:54:12.068+0000] {processor.py:157} INFO - Started process (PID=39458) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:12.071+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:54:12.073+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:12.073+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:12.118+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:12.523+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:12.523+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:54:12.572+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:12.572+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:54:12.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.553 seconds
[2023-10-08T02:54:42.782+0000] {processor.py:157} INFO - Started process (PID=39538) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:42.784+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:54:42.787+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:42.786+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:42.817+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:42.860+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:42.860+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:54:42.908+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:42.908+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:54:42.944+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.168 seconds
[2023-10-08T02:54:47.322+0000] {processor.py:157} INFO - Started process (PID=39553) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:47.325+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:54:47.327+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:47.327+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:47.368+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:47.422+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:47.422+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:54:47.479+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:47.478+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:54:47.523+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.208 seconds
[2023-10-08T02:54:48.554+0000] {processor.py:157} INFO - Started process (PID=39555) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:48.558+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:54:48.562+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:48.561+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:48.690+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:54:48.744+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:48.743+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:54:48.811+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:54:48.811+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:54:48.851+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.307 seconds
[2023-10-08T02:55:16.150+0000] {processor.py:157} INFO - Started process (PID=39631) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:16.152+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:55:16.155+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:55:16.154+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:16.208+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:16.256+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:55:16.255+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:55:16.311+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:55:16.311+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:55:16.365+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.222 seconds
[2023-10-08T02:55:47.122+0000] {processor.py:157} INFO - Started process (PID=39712) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:47.126+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:55:47.130+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:55:47.129+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:47.186+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:55:47.177+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 19, in <module>
    staging_layer = staging_tasks()
  File "/opt/airflow/dags/groups/group_staging_layer.py", line 76, in staging_tasks
    autocommit=True
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/microsoft/mssql/operators/mssql.py", line 60, in __init__
    super().__init__(conn_id=mssql_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 231, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 133, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 806, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 227, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'staging_layer.insert_into_stg_actor_film_assoc' has already been added to the DAG
[2023-10-08T02:55:47.190+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:55:47.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.123 seconds
[2023-10-08T02:56:17.440+0000] {processor.py:157} INFO - Started process (PID=39793) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:17.444+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:56:17.447+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:56:17.447+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:17.497+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:17.999+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:56:17.999+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:56:18.066+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:56:18.066+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:56:18.126+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.696 seconds
[2023-10-08T02:56:48.579+0000] {processor.py:157} INFO - Started process (PID=39878) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:48.582+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:56:48.585+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:56:48.585+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:48.651+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:56:48.637+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/group_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag.py", line 19, in <module>
    staging_layer = staging_tasks()
  File "/opt/airflow/dags/groups/group_staging_layer.py", line 93, in staging_tasks
    autocommit=True
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/microsoft/mssql/operators/mssql.py", line 60, in __init__
    super().__init__(conn_id=mssql_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 231, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 133, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 429, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 806, in __init__
    task_group.add(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/task_group.py", line 227, in add
    raise DuplicateTaskIdFound(f"{node_type} id '{key}' has already been added to the DAG")
airflow.exceptions.DuplicateTaskIdFound: Task id 'staging_layer.insert_into_stg_actor' has already been added to the DAG
[2023-10-08T02:56:48.655+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:56:48.738+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.170 seconds
[2023-10-08T02:57:19.100+0000] {processor.py:157} INFO - Started process (PID=39958) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:19.108+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:57:19.119+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:19.118+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:19.191+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:19.780+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:19.780+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:57:19.848+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:19.848+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:57:19.909+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.825 seconds
[2023-10-08T02:57:50.100+0000] {processor.py:157} INFO - Started process (PID=40039) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:50.104+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:57:50.108+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:50.107+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:50.164+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:57:50.238+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:50.238+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:57:50.545+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:57:50.545+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:57:50.620+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.543 seconds
[2023-10-08T02:58:21.318+0000] {processor.py:157} INFO - Started process (PID=40124) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:21.322+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:58:21.326+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:21.326+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:21.376+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:22.088+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:22.087+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:58:22.159+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:22.158+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-07T00:00:00+00:00, run_after=2023-10-08T00:00:00+00:00
[2023-10-08T02:58:22.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.932 seconds
[2023-10-08T02:58:53.684+0000] {processor.py:157} INFO - Started process (PID=40211) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:53.695+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:58:53.718+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:53.717+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:53.849+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:58:53.999+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:53.998+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:58:54.171+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:58:54.171+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T02:58:54.282+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.644 seconds
[2023-10-08T02:59:40.628+0000] {processor.py:157} INFO - Started process (PID=40280) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T02:59:40.648+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T02:59:40.697+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:59:40.695+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T02:59:40.860+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T02:59:41.011+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:59:41.010+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T02:59:41.449+0000] {logging_mixin.py:150} INFO - [2023-10-08T02:59:41.448+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T02:59:42.421+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 1.859 seconds
[2023-10-08T03:00:12.773+0000] {processor.py:157} INFO - Started process (PID=40362) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:12.776+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:00:12.781+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:12.780+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:12.832+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:12.888+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:12.888+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:00:12.958+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:12.957+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:00:13.007+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.242 seconds
[2023-10-08T03:00:43.106+0000] {processor.py:157} INFO - Started process (PID=40442) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:43.108+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:00:43.110+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:43.109+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:43.134+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:00:43.165+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:43.165+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:00:43.314+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:00:43.314+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:00:43.337+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.237 seconds
[2023-10-08T03:01:14.172+0000] {processor.py:157} INFO - Started process (PID=40531) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:14.175+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:01:14.178+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:14.178+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:14.211+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:14.256+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:14.256+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:01:14.307+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:14.307+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:01:14.339+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.174 seconds
[2023-10-08T03:01:44.719+0000] {processor.py:157} INFO - Started process (PID=40612) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:44.723+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:01:44.728+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:44.727+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:44.827+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:01:44.943+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:44.943+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:01:45.066+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:01:45.066+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:01:45.130+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.422 seconds
[2023-10-08T03:02:19.114+0000] {processor.py:157} INFO - Started process (PID=40688) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:02:19.155+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:02:19.181+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:02:19.179+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:02:19.811+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:02:21.145+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:02:21.100+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:02:22.399+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:02:22.398+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:02:22.776+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 3.737 seconds
[2023-10-08T03:03:01.841+0000] {processor.py:157} INFO - Started process (PID=40765) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:01.853+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:03:01.864+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:01.861+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:01.973+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:02.083+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:02.082+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:03:02.311+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:02.311+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:03:02.494+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.671 seconds
[2023-10-08T03:03:32.988+0000] {processor.py:157} INFO - Started process (PID=40839) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:32.991+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:03:32.993+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:32.993+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:33.028+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:03:33.075+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:33.075+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:03:33.125+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:03:33.124+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:03:33.156+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.175 seconds
[2023-10-08T03:04:03.278+0000] {processor.py:157} INFO - Started process (PID=40921) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:03.280+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:04:03.282+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:03.282+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:03.311+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:03.347+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:03.346+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:04:03.382+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:03.382+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:04:03.405+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.132 seconds
[2023-10-08T03:04:33.476+0000] {processor.py:157} INFO - Started process (PID=41010) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:33.478+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:04:33.480+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:33.479+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:33.504+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:04:33.541+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:33.541+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:04:33.575+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:04:33.575+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:04:33.600+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.129 seconds
[2023-10-08T03:05:03.782+0000] {processor.py:157} INFO - Started process (PID=41088) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:03.784+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:05:03.786+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:03.786+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:03.814+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:03.851+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:03.850+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:05:03.893+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:03.893+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:05:03.928+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.152 seconds
[2023-10-08T03:05:34.338+0000] {processor.py:157} INFO - Started process (PID=41177) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:34.340+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:05:34.343+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:34.342+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:34.374+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:05:34.416+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:34.416+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:05:34.463+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:05:34.463+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:05:34.495+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.162 seconds
[2023-10-08T03:06:04.901+0000] {processor.py:157} INFO - Started process (PID=41257) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:04.903+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:06:04.905+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:04.905+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:04.930+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:04.972+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:04.971+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:06:05.013+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:05.012+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:06:05.045+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.149 seconds
[2023-10-08T03:06:35.359+0000] {processor.py:157} INFO - Started process (PID=41346) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:35.362+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:06:35.364+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:35.363+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:35.388+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:06:35.426+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:35.426+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:06:35.465+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:06:35.465+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:06:35.493+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.139 seconds
[2023-10-08T03:07:05.569+0000] {processor.py:157} INFO - Started process (PID=41426) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:05.572+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:07:05.574+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:05.573+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:05.597+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:05.633+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:05.633+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:07:05.676+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:05.676+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:07:05.706+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.141 seconds
[2023-10-08T03:07:35.763+0000] {processor.py:157} INFO - Started process (PID=41515) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:35.765+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:07:35.767+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:35.766+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:35.795+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:07:35.835+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:35.835+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:07:35.880+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:07:35.880+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:07:35.911+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.153 seconds
[2023-10-08T03:08:05.955+0000] {processor.py:157} INFO - Started process (PID=41596) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:05.957+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:08:05.959+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:05.959+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:05.991+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:06.035+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:06.035+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:08:06.079+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:06.079+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:08:06.109+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.162 seconds
[2023-10-08T03:08:36.336+0000] {processor.py:157} INFO - Started process (PID=41684) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:36.339+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:08:36.341+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:36.340+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:36.370+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:36.422+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:36.421+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:08:36.469+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:36.468+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:08:36.535+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.204 seconds
[2023-10-08T03:08:59.555+0000] {processor.py:157} INFO - Started process (PID=41748) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:59.558+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:08:59.561+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:59.560+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:59.616+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:08:59.653+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:59.653+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:08:59.706+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:08:59.706+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:08:59.751+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.202 seconds
[2023-10-08T03:09:07.856+0000] {processor.py:157} INFO - Started process (PID=41768) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:07.858+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:09:07.860+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:07.860+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:07.900+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:07.946+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:07.946+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:09:07.994+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:07.994+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:09:08.031+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.182 seconds
[2023-10-08T03:09:38.505+0000] {processor.py:157} INFO - Started process (PID=41855) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:38.511+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:09:38.513+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:38.513+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:38.569+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:09:38.630+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:38.629+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:09:38.695+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:09:38.695+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:09:38.745+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.247 seconds
[2023-10-08T03:10:09.028+0000] {processor.py:157} INFO - Started process (PID=41936) to work on /opt/airflow/dags/group_dag.py
[2023-10-08T03:10:09.033+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/group_dag.py for tasks to queue
[2023-10-08T03:10:09.036+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:10:09.036+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag.py
[2023-10-08T03:10:09.113+0000] {processor.py:836} INFO - DAG(s) dict_keys(['group_dag']) retrieved from /opt/airflow/dags/group_dag.py
[2023-10-08T03:10:09.220+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:10:09.220+0000] {dag.py:2742} INFO - Sync 1 DAGs
[2023-10-08T03:10:09.314+0000] {logging_mixin.py:150} INFO - [2023-10-08T03:10:09.314+0000] {dag.py:3508} INFO - Setting next_dagrun for group_dag to 2023-10-08T00:00:00+00:00, run_after=2023-10-09T00:00:00+00:00
[2023-10-08T03:10:09.378+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/group_dag.py took 0.360 seconds
